storage/volumes.md

```diff
@@ -512,12 +518,92 @@ $ docker volume create \
 	--name cif-volume
 ```
 
-Notice the `addr` option is required if using a hostname instead of an IP so docker can perform the hostname lookup.
+The `addr` option is required if you specify a hostname instead of an IP.
+This lets Docker perform the hostname lookup.
+
+### Mount a block storage device
+
+You can mount a block storage device, such as an external drive or a drive partition, to a container.
+The following example shows how to create and use a file as a block storage device,
+and how to mount the block device as a container volume.
+
+> **Important**
+>
+> The following procedure is only an example.
+> The solution illustrated here isn't recommended as a general practice.
+> Don't attempt this approach unless you're very confident about what you're doing.
+{: .important }
+
+Under the hood, the `--mount` flag using the `local` storage driver invokes the
+Linux `mount` syscall and forwards the options you pass to it unaltered.
+Docker doesn't implement any additional functionality on top of the native mount features supported by the Linux kernel.
+
+If you're familiar with the
+[Linux `mount` command](https://man7.org/linux/man-pages/man8/mount.8.html),
+you can think of the `--mount` options being forwarded to the `mount` command as follows:
+
+```console
+$ mount -t <mount.volume-opt.type> <mount.volume-opt.device> <mount.dst> -o <mount.volume-opts.o>
+```
+
+In other words, the following commands are equivalent:
```

Comment:
"Equivalent" is a bit too strong IMO. It sounds like these two commands would produce exactly the same result, but they don't.
First runs a container that mounts it, and second mount it on the host.

It would be more correct to say, that the from the container perspective, docker run example sets up the mount in a way that executing "mount -t ext4 /dev/loop5 /external-drive" inside container would (if the container had direct access to /dev/loop5, but it doesn't, so you must rely on `docker run` to do that).